### PR TITLE
replace print statements with logging - synapse/store

### PR DESCRIPTION
**Pull request by sujaypatil96**
This PR works to replace the `print` statements in the `synapse/store` module with `logging` statements.

_Note_: I'm planning to address issue #317 in a series of multiple PRs, where I'll refactor schematic, module-by-module. This is the first of those PRs.

I've also been meaning to make this addition to the `config.yml` file:

```
manifest:
  synapse:
    title: 'Patient_Manifest'
    data_type: 'Patient'
  local:
    path: '/path/to/local/manifest'
```

Just a note, we can consider this if we want to allow users to configure the path to the local manifest. But I guess we can talk more about it when working on issue #335.

### CODE REVIEW

File: schematic/exceptions.py [hunk 1]
```diff
@@ -53,3 +53,28 @@ def __init__(self, arg_name: str,
...
+        message: custom/pre-defined error message to be returned.
+
+    Returns:
+        message.
+    """
+
+    def __init__(self, project: str, message: str = None) -> str:
+        if message is not None:
+            self.message = message
+        else:
```
Comment (BrunoGrandePhD) [inline schematic/exceptions.py:72]: You can avoid this else statement by setting the message to the default value before the if statement and then overwrite the message with user-provided value `if message:`.

File: schematic/exceptions.py [hunk 2]
```diff
@@ -53,3 +53,28 @@ def __init__(self, arg_name: str,
...
+            self.message = (
+                f"Your access to '{project}'' could not be resolved. "
+                "Please check your credentials and try again."
+            )
+        super().__init__(self.message)
+
+    def __str__(self):
```
Comment (BrunoGrandePhD) [inline schematic/exceptions.py:79]: If this pattern (and the pattern in __init__) is going to repeat itself for each exception, you could look into creating a base `SchematicException` class.

**Review by BrunoGrandePhD — APPROVED** [review]
I'm happy with the PR as is, but feel free to implement some of my suggested tweaks. 

🚀

File: schematic/exceptions.py (hunk 1 above)
Comment (sujaypatil96) [reply schematic/exceptions.py]: Great. This is cleaner, yes. I've incorporated this change.

File: schematic/exceptions.py (hunk 2 above)
Comment (sujaypatil96) [reply schematic/exceptions.py:82]: Thanks for the tip Bruno. I'll work on this and incorporate in a future PR.

**Pull request merged**